internal/controller/numaresourcesoperator_controller_test.go

```diff
@@ -319,6 +321,19 @@ var _ = Describe("Test NUMAResourcesOperator Reconcile", func() {
 					Expect(*ds.Spec.Template.Spec.Affinity.PodAntiAffinity).To(Equal(expected))
 				})
 
+				It("should have a update strategy with MaxUnavailable set", func() {
+					dsKey := client.ObjectKey{
+						Name:      objectnames.GetComponentName(nro.Name, pn1),
+						Namespace: testNamespace,
+					}
+					var ds appsv1.DaemonSet
+					Expect(reconciler.Client.Get(context.TODO(), dsKey, &ds)).To(Succeed())
+
+					Expect(ds.Spec.UpdateStrategy.Type).To(Equal(appsv1.RollingUpdateDaemonSetStrategyType))
+					Expect(ds.Spec.UpdateStrategy.RollingUpdate).ToNot(BeNil())
+					Expect(ds.Spec.UpdateStrategy.RollingUpdate.MaxUnavailable.String()).To(BeStringPercentageAtLeast(10))
```

Comment:
de we still intend this to be 10? and considering the logic of `MaxUnavailable` shouldn't this be "at most" instead of "at least"?

Reply:
The intent here is to set (and check) we do set a `MaxUnavaialble` whose value is 10% or greater, do you think this is expressed clearly enough from the testcase?